src/components/CubePreview.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Card, CardHeader } from 'reactstrap';
+
+
+class CubePreview extends React.Component {
```

Comment:
Components with no state and no real instance methods should be written as functional components:

```javascript
const CubePreview = ({ cube }) =>
  <Card>
    {/* ... */}
  </Card>
```

Reply:
You can move the get_cube_id method outside the component then.